doc/lua_api.txt

```diff
@@ -3235,6 +3247,20 @@ These functions return the leftover itemstack.
 Class reference
 ---------------
 
+### ModChannel
+
+An interface to use mod channels on client and server
+
+#### Methods
+* `leave()`: leave the mod channel.
+    * Server leaves channel `channel_name`.
+    * No more incoming or outgoing messages can be sent to this channel from server mods.
+    * This invalidate all future object usage
+    * Ensure your set mod_channel to nil after that to free Lua resources
+* `is_writeable()`: returns true if channel is writeable and mod can send over it.
```

Comment:
isnt this always true for the server-side?

Reply:
read only is not totally implemented but no it's not always true server side, if you leave channel and call is_writeable false is returned